bedrock/firefox/templates/firefox/os/mwc-2015-preview.html

```diff
@@ -0,0 +1,180 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Blaze Your Own Path - Mozilla at MWC 2015')}}{% endblock %}
+
+{% block body_id %}firefox-os{% endblock %}
+{% block body_class %}firefox-os{% endblock %}
+
+{% block site_header_nav %}{% endblock %}
+{% block site_header_logo %}{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_os_mwc_2015_preview' %}
+  <!--[if lt IE 8]>
+  {% stylesheet 'firefox_os_mwc_2015_preview_ie7' %}
+  <![endif]-->
+{% endblock %}
+
+{% block content %}
+<main role="main">
+  <section id="intro">
+    <div class="container">
+      <h2 class="logo">
+        {{ high_res_img('img/firefox/os/mwc-2015-preview/firefox-os.png', {'alt': 'Firefox OS', 'width': '224', 'height': '70'}) }}
+      </h2>
+
+      <div id="intro-copy">
+        <header>
+          <h1>{{ _('Unleash the future') }}</h1>
+          <h2>{{ _('Mozilla at Mobile World Congress 2015') }}</h2>
+        </header>
+
+        <p>
+          {{ _('Since launching just two short years ago, Firefox OS has gone through some dramatic growth and exciting changes.') }}
+          {{ _('Now in over 30 countries around the globe, Firefox OS devices are available from entry level handsets to mid-tier smartphones manufactured by top-end partners.') }}
+        </p>
+
+        <p>
+          {{ _('For 2015, we’re going to continue to build on what we’ve started.') }}
+          {{ _('We’ve seen 50+ launches since July 2013 and things are about to get even more exciting.') }}
+          {{ _('We’re introducing new chipsets in India and the Philippines, continuing to build the Firefox Marketplace with a bigger ecosystem and even more content, and with our partner Panasonic, we’re about to release a line of Firefox OS-powered televisions out into the world.') }}
+        </p>
+
+        <p>
+          {{ _('Come and partner with Firefox OS — the most exciting new platform in mobile.') }}
+        </p>
+
+        {{ high_res_img('img/firefox/os/mwc-2015-preview/mwc-logo.png', {'alt': 'Mobile World Congress', 'width': '168', 'height': '68'}) }}
+      </div>
+      <figure id="intro-phone">
```

Comment:
If I'm being super nitpicky, I'd argue this isn't a semantically appropriate use of the <code>figure</code> element and would switch it to a plain <code>div</code>, and I'd give the image an empty <code>alt</code> attribute since it's essentially presentational.